pkg/cmd/show.go

```diff
@@ -245,17 +249,10 @@ func showVpnShoot(targetReader TargetReader) {
 func showPrometheus(targetReader TargetReader) {
 	username, password = getMonitoringCredentials()
 	showPod("prometheus", "seed", targetReader)
-	output, err := ExecCmdReturnOutput("kubectl", "--kubeconfig="+KUBECONFIG, "get", "ingress", "prometheus", "-n", getTechnicalID())
+	KUBECONFIG := getKubeConfigOfClusterType("seed")
+	url, err := ExecCmdReturnOutput("kubectl", "--kubeconfig="+KUBECONFIG, "get", "ingress", "prometheus", "-n", getFromTargetInfo("shootTechnicalID"), "--no-headers", "-o", "custom-columns=:spec.rules[].host")
 	if err != nil {
-		fmt.Println("Cmd was unsuccessful")
-		os.Exit(2)
-	}
-	list := strings.Split(output, " ")
-	url := "-"
-	for _, val := range list {
-		if strings.HasPrefix(val, "p.") {
-			url = val
-		}
+		log.Fatalf("Cmd was unsuccessful")
```

Comment:
Why have you removed `os.Exit(2)`?

Reply:
oh, con't remeber,  I maybe change it at that time.The `log.Fatalf` also provider os.Exit(1) and Printtf at the same time as I check. should I change back to os.Exit(2)?

`Fatalf is equivalent to Printf() followed by a call to os.Exit(1).`

Reply:
Ah ok, wasn't aware of that. It's ok